EmployeeManagementSystem/DAL/IDAL.cs

```diff
@@ -0,0 +1,12 @@
+using System.Collections.Generic;
+
+namespace EmployeeManagement;
+
+public interface IDAL
```

Comment:
1. Name this as IEmployeeDAL
2. Update method we don't need multiple employees. Just pass employee to be updated
3. Insert, Update, Delete > please use proper return type. void is not good return type
4. Get >> Change it to GetAll
5. Remove GetFilePath method. I don't know why do we need that.